queries.sql

```diff
@@ -38,9 +38,36 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
-
-
-
-
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+SELECT a.name
+FROM animals a
+JOIN species s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
+SELECT o.full_name, a.name
+FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id;
+SELECT s.name, COUNT(*) AS animal_count
+FROM animals a
+JOIN species s ON a.species_id = s.id
+GROUP BY s.name;
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+JOIN species s ON a.species_id = s.id
+WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+LEFT JOIN escape_attempts e ON a.id = e.animal_id
+WHERE o.full_name = 'Dean Winchester' AND e.animal_id IS NULL;
```

Comment:
- [ ] Kindly fix the `List all animals owned by Dean Winchester that haven't tried to escape` query as there is no table called `escape_attempts`. It's only a column on the animal's table.